docs/data/toolpad/connecting-to-datasources/queries.md

```diff
@@ -1,16 +1,16 @@
 # Queries
 
-<p class="description">Toolpad allows you to bring remote data as state to the page and allows your UI to bind itself to this state.</p>
+<p class="description">A query allows to bring remote data as "state" to the page and allows your UI to bind itself to that state.</p>
 
-Be it a database table or an external API, most data we want to display in Toolpad pages originates serverside. Toolpad offers mechanisms to bring this serverside data to the page. The most common way is through **queries**. To configure a query we need to follow these steps:
```

Comment:
The use of "we" is rarely appropriate.

Reply:
alternatively

"most data displayed on toolpad pages"

"to configure a query, follow these steps"